DigitalLearningSolutions.Data/DataServices/UserDataService/UserDataService.cs

```diff
@@ -24,6 +24,8 @@ public interface IUserDataService
             int categoryId
         );
 
+        void UpdateAdminUserFailedLoginCount(int adminId, int updatedCount);

```

Comment:
Is this intended addition?

Reply:
This is used for the unlock admin account functionality that was merged to master, the merge of master introduced it to this branch.